CHANGELOG.md

```diff
@@ -0,0 +1,130 @@
+  @dhoppe. Election officers were @roidelapluie and @danney.
+
+
+## September 2016
+
+* New Puppet modules: puppet-boolean, puppet-sftp_jail, puppet-mrepo, puppet-php
+* New repository: ra10ke
+
+
+## August 2016
+
+* Governance document is merged (#45)
+* New mailing list: voxpupuli@groups.io
+* New Puppet modules: puppet-googleauthenticator, puppet-selinux, puppet-nginx,
+  puppet-fetchcrl, puppetlabs-splunk
+
+## July 2016
+
+* New Puppet module: puppet-rhsm
+
+
+## June 2016
+
+* New Puppet modules: puppet-gerrit, puppetlabs-drbd
+* New repository: jenkins-puppet-syntax-check
+
+
+## May 2016
+
+* New Puppet module: puppet-puppetserver
+* New repository: vagrant-librarian-puppet
+
+
+## April 2016
+
+* New Puppet modules: puppet-zabbix, puppet-squid, puppet-hiera,
+  puppetlabs-dhcp, puppetlabs-bacula
+
+
+## March 2016
+
+* New repositories: librarian, hiera-file
+
+
+## February 2016
+
+* New Puppet modules: puppet-mumble, puppet-minecraft, puppet-ghost,
+* New Puppet Lint plugin: puppet-lint-resource_reference_syntax
+* New repository: vim-puppet
+
+
+## January 2016
+
+* New name (Vox Pupuli) and new website
+* New Puppet module: puppet-filemapper
+
+
+## December 2015
+
+* New Puppet modules: puppet-blacksmith, puppet-etherpad,
+  puppet-community-open-prs, puppet-gluster
+
+
+## August 2015
+
+* New Puppet module: puppet-module-collectd
+
+
+## July 2015
+
+* New Puppet module: puppet-git
+
+
+## June 2015
+
+* New Puppet module: puppet-mysql_java_connector
+
+
+## May 2015
+
+* New Puppet modules: puppetlabs-nodejs, puppetlabs-corosync
+
+
+## April 2015
+
+* New Puppet modules: puppet-jira, puppet-staging, puppet-archive, puppet-iis,
+  puppet-windowsfeature, puppet-download_file
+* New repository: modulesync
+
+
+## March 2015
+
+* New Puppet modules: puppet-rundeck, puppet-kafka, puppet-jenkins_job_builder,
+  puppet-graphite_powershell, puppet-sslcert, puppet-nsclient,
+  puppet-visualstudio, puppet-confluence, puppet-msoffice, puppet-dotnet,
+  puppet-windows_power, puppet-stash, puppet-windows_eventlog,
+  puppet-windows_firewall, puppet-windows_autoupdate, puppetlabs-mcollective
+* New Puppet Lint plugin: puppet-lint-appends-check
+* New repository: metadata-json-lint
+
+
+## February 2015
+
+* New Puppet Lint plugins: puppet-lint-file_ensure-check,
+  puppet-lint-absolute_classname-check, puppet-lint-leading_zero-check,
+  puppet-lint-spaceship_operator_without_tag-check,
+  puppet-lint-trailing_comma-check, puppet-lint-undef_in_function-check,
+  puppet-lint-version_comparison-check, puppet-lint-unquoted_string-check,
+  puppet-lint-empty_string-check
+* New repositories: pcci, pcci-configuration
+
+
+## November 2014
+
+* New Puppet module: puppet-network
+
+
+## October 2014
+
+* New Puppet module: puppet-alternatives
+* New repositories: puppetboard, pypuppetdb
+
+
+## September 2014
+
+* New Puppet modules: puppet-module-amanda, puppet-module-puppetboard,
```

Comment:
we renamed the modules to puppet-amanda and puppet-puppetboard. not sure if we want to list the original or current name.